### PR TITLE
Feature/logging

### PR DESCRIPTION
**Pull request by arbulu89**
Add logging system to daemon or background execution.

Adding this new entries to the configuration file will change the logging system and use the configuration file instead. There, we will configure the output file.

By defaul, INFO level logged to /var/log/hanadb_exporter.log
And ERROR and WARN to the syslog

### CODE REVIEW

File: logging_config.ini [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+handlers=fileHandler,sysHandler
+
+[logger_fileLogger]
+level=INFO
+handlers=fileHandler
+qualname=shaptools,hanadb_exporter
+propagate=0
+
+[logger_sysLogger]
+level=WARNING
```
Comment (diegoakechi) [inline logging_config.ini]: @arbulu89 By default, I would forward Warning and Errors to the log file too. It is easier to debug when it is all in the same file. I would also set Warn and Error only as the default, as the INFO is really verbose.

Other point is that you expect this file to be configurable (as it is being installed on /etc), so I would recommend to include comment headers on each section in order to instruct the users on how to use it.

Comment (arbulu89) [reply logging_config.ini]: @diegoakechi Yes, right now up to INFO (what means, CRITICAL, ERROR, WARNING, INFO) are written in the log file and only up to WARNING in syslog. It's true, that it is quite verbose, we can reduce the level by default.

Ok, i will include some comments to explain how the configuration file works. Anyway, this is a standard python logging configuration file, so documentation is easy to find

Comment (diegoakechi) [reply logging_config.ini]: @arbulu89 Yes, but still I think some comments is nice to have and maybe a link pointing to more information from the official documentation.

Comment (arbulu89) [reply logging_config.ini]: Working on that

**Pull request merged**